Dockerfile

```diff
@@ -1,3 +1,11 @@
+# Build Container
+FROM golang:buster AS build-env
+
+ADD . /workdir
```

Comment:
COPY is preferable.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy